packages/types/src/element/textElement.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import type { Element } from './element';
+import type { Pattern } from './pattern';
+
+export type FontStyle = 'normal' | 'italic';
```

Comment:
Note: Seeing `regular` in use as well in the codebase.

Reply:
Oh yeah, we're using both `'regular'` (in text panel and templates) and `'normal'` (in the design panel and design menu). Hmm, what to do? Should we make these aligned or?

Reply:
Sounds like another follow-up issue if these are the same things :) Until then I suppose we should add `regular` to the union type.

Reply:
Fixed in 0a8dbd2af8. New issue in https://github.com/GoogleForCreators/web-stories-wp/issues/12333.